src/services/types.ts

```diff
@@ -684,6 +684,13 @@ export interface LanguageService {
     getSupportedCodeFixes(fileName?: string): readonly string[];
 
     dispose(): void;
+    getPasteEdits(
+        targetFile: string,
+        copies: { text: string; range?: { file: string; range: TextRange; }; }[],

```

Comment:
It might be worth extracting this object type to an interface. Then you can avoid duplicating it in the fourslash code.